.drone.yml

```diff
@@ -0,0 +1,49 @@
+kind: pipeline
+type: docker
+name: default
+steps:
+  - name: restore-cache
+    image: drillster/drone-volume-cache
+    settings:
+      restore: true
+      mount:
+        - ./vendor
+    volumes:
+      - name: cache
+        path: /cache
+  
+  - name: build
+    image: registry.gitlab.com/fun-tech/fundraising-frontend-docker:ci
+    environment:
+      COMPOSER_CACHE_DIR: /composer_cache
+      GITHUB_TOKEN: 
+        from_secret: github_token
+    volumes:
+      - name: cache
+        path: /composer_cache
+    commands:
+      # Environment setup
+      - echo -e "machine github.com\n  login $GITHUB_TOKEN" > ~/.netrc
```

Comment:
I wonder if we can use something like this to auto make issues in the content repo drone setup?